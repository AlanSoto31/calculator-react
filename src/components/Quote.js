const Quote = () => (
  <p className="text-quote">
    Mathematics is not about numbers, equations, computations,
    or algorithms: it is about understanding. â€“
    <q>William Paul Thurston.</q>
  </p>
);

export default Quote;
