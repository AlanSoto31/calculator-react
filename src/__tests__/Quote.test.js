import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('Quote component renders', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
