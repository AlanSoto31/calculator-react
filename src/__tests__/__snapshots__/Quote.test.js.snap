// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders 1`] = `
<p
  className="text-quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“
  <q>
    William Paul Thurston.
  </q>
</p>
`;
